.pre-commit-config.yaml

```diff
@@ -0,0 +1,34 @@
+repos:
+# Need to drop python 3.5 and 3.6 before we include python upgrade
+#  - repo: https://github.com/asottile/pyupgrade
+#    rev: v2.31.0
+#    hooks:
+#      - id: pyupgrade
+#        args: ["--py37-plus"]
+
+  - repo: https://github.com/adamchainz/django-upgrade
+    rev: '1.4.0'
+    hooks:
+      - id: django-upgrade
+        args: [--target-version, "1.11"]
```

Comment:
Changed via a GH suggestion.

Reply:
Changing the hook to 2.2 breaks the code because it makes it django 2.2 compatible. You are going to get a lot of complaints as this is added to more projects, especially when enterprise is behind always.